crates/cli-flags/src/lib.rs

```diff
@@ -380,6 +380,10 @@ wasmtime_option_group! {
         pub wide_arithmetic: Option<bool>,
         /// Configure support for the extended-const proposal.
         pub extended_const: Option<bool>,
+        /// Configure support for the exceptions proposal.
+        pub exceptions: Option<bool>,
+        /// Configure support for the legacy exceptions proposal.
+        pub legacy_exceptions: Option<bool>,
```

Comment:
This should be `doc(hidden)` and `clap(hidden)` or whatever as well, same as the `Config::method`.

Reply:
We unfortunately don't have that here, but I can put "DEPRECATED" in the doc at least!